include/main.h

```diff
@@ -1,32 +1,32 @@
 
-/** Simple program "search and sort algorithms" realized.  
+/** @brief Simple program "search and sort algorithms" realized.  
  * @file main.h
```

Comment:
@brief hilft, dass auch in der Übersicht die nachfolgenden Worte angezeigt werden. nice feature...